### ISSUE TITLE
[1.12.2 Feature Request] Growthcraft Cellar: Adding more Booze

### ISSUE THREAD

**Issue by Cinamonfire** [GrowthcraftCE/Growthcraft-1.12#197]
Ok since Binnies Mods/ Forestry Are Being a pain in making booze and spirits (because they require RF to function and make only half of the drinks available with the glasses now disabled), and Rustics development being kinda slow, I Kinda gave in to giving GrowthcraftCE a shot in making More Features for alcohol options. And IMO Growthcraft are pretty damn good at simplifying booze making as a mod concept unlike psychedelicraft which has a long fermentation time and distillering time. I came up with a Feature Request that the team may be Interested in...

**Alex's And Steve's May I Proudly Present:**

### GROWTHCRAFT: SPIRITS

Growthcraft: Spirits is a new idea in adding more potent booze into the game with 5 different new booze types dubbed spirits, unlike ale, wine, cider, mead, sake and kumis that requires only one crafting procedure, Spirits require two or three!

Example in turning wheat wash into Whiskey:
1.
First you need Wheat Wash made in the brew kettle and with the wheat wash you basically have to distill it on a distillery block which is fueled like the brew kettle under an open fire (or lava) .
2.
With the wheat wash in the distillery it takes 24,000 ticks to be distilled into whats called wheat wash[Distilled] where you gain a 1 minecraft bucket of wheat wash: a fourth bucket of distilled wheat wash Ratio.
3.
Once the Distilled Wheat Wash is ready you either pump it or carry it in a bucket to the Growthcraft Cellars Barrels to ferment the Distilled wheat wash with yeast, nether wart, or you can add Fire yeast (which is yeast made with blaze powder) to make Fire-Water (whiskey variant) or Sweet yeast (which is yeast made with honey) to make Honeyjack (another variant)
And After 48,000 ticks (2 minecraft days) Your Whiskey/Fire-Water/Honeyjack is Ready!

Bonus 4:
when a spirit is drunk and its parent ingredient for say ale gives you some sort of potion effect , the spirit gives an extra one related to it but in exchange drinking a spirit increases your tipsy level by two instead of one. Example: Whiskey gives the same effect as ale but gives regeneration 1 as well, Fire-Water gives fire resistance and Honeyjack is a mix of ale and mead!

My 4 other Spirit ideas are made on the same aspect of the brew kettle/distillery procedure but with new ingredients: 
Vodka = Potato Wash > Distillery > Barrel (Regular vodka, Apple vodka, Poitin)
Rum = Sugar Wash > Distillery > Barrel (Regular Rum, redistill for white rum, fruit rum)
Brandy = Red Grape Wine > Distillery > Barrel (Regular Brandy, Cognac, Fruit Brandy)
Tequila = Cactus Wash > Distillery > Barrel (Regular Tequila, Sweet tequila, Dry Tequila)

I hope this will be in game someday :3

**Comment by Alatyami** [comment]
Definitely, we already have distilling on the backlog.  And adding additional fermentables is trivial.

**Comment by Cinamonfire** [comment]
> Definitely, we already have distilling on the backlog. And adding additional fermentables is trivial.

I cant see it in the backlog though XD
